CMakeLists.txt

```diff
@@ -142,70 +142,70 @@ set(EXAMPLE_SOURCE_FILES
     examples/compact.c
     examples/edge.c)
 set(OTHER_SOURCE_FILES
-    src/apps/filters/h3ToGeo.c
+    src/apps/filters/cellToPoint.c
     src/apps/filters/h3ToLocalIj.c
     src/apps/filters/localIjToH3.c
     src/apps/filters/h3ToComponents.c
-    src/apps/filters/geoToH3.c
-    src/apps/filters/h3ToGeoBoundary.c
-    src/apps/filters/kRing.c
-    src/apps/filters/hexRange.c
+    src/apps/filters/pointToCell.c
+    src/apps/filters/cellToBoundary.c
+    src/apps/filters/gridDisk.c
+    src/apps/filters/gridDiskUnsafe.c
     src/apps/testapps/testVertexGraph.c
-    src/apps/testapps/testCompact.c
-    src/apps/testapps/testPolyfill.c
-    src/apps/testapps/testPolyfillReported.c
+    src/apps/testapps/testCompactCells.c
+    src/apps/testapps/testPolygonToCells.c
+    src/apps/testapps/testPolygonToCellsReported.c
     src/apps/testapps/testPentagonIndexes.c
-    src/apps/testapps/testKRing.c
-    src/apps/testapps/testH3ToGeoBoundary.c
-    src/apps/testapps/testH3ToParent.c
+    src/apps/testapps/testGridDisk.c
+    src/apps/testapps/testCellToBoundary.c
+    src/apps/testapps/testCellToParent.c
     src/apps/testapps/testH3Index.c
     src/apps/testapps/mkRandGeoBoundary.c
-    src/apps/testapps/testGeoToH3.c
+    src/apps/testapps/testPointToCell.c
     src/apps/testapps/testH3NeighborRotations.c
-    src/apps/testapps/testMaxH3ToChildrenSize.c
-    src/apps/testapps/testHexRanges.c
-    src/apps/testapps/testH3ToGeo.c
-    src/apps/testapps/testH3ToCenterChild.c
-    src/apps/testapps/testH3ToChildren.c
-    src/apps/testapps/testH3GetFaces.c
-    src/apps/testapps/testGeoCoord.c
-    src/apps/testapps/testHexRing.c
+    src/apps/testapps/testMaxCellToChildrenSize.c
+    src/apps/testapps/testGridDisksUnsafe.c
+    src/apps/testapps/testCellToPoint.c
+    src/apps/testapps/testCellToCenterChild.c
+    src/apps/testapps/testCellToChildren.c
+    src/apps/testapps/testGetIcosahedronFaces.c
+    src/apps/testapps/testGeoPoint.c
+    src/apps/testapps/testGridRingUnsafe.c
     src/apps/testapps/testH3SetToVertexGraph.c
     src/apps/testapps/testBBox.c
     src/apps/testapps/testVertex.c
     src/apps/testapps/testPolygon.c
     src/apps/testapps/testVec2d.c
     src/apps/testapps/testVec3d.c
-    src/apps/testapps/testH3UniEdge.c
-    src/apps/testapps/testH3UniEdgeExhaustive.c
+    src/apps/testapps/testDirectedEdge.c
+    src/apps/testapps/testDirectedEdgeExhaustive.c
     src/apps/testapps/testLinkedGeo.c
     src/apps/testapps/mkRandGeo.c
     src/apps/testapps/testH3Api.c
     src/apps/testapps/testH3SetToLinkedGeo.c
     src/apps/testapps/testH3ToLocalIj.c
```

Comment:
I couldn't find any evidence anywhere that it was being renamed or having its experimental status removed, so that's why I didn't touch it in this PR. This *is* just part 1, so we can revisit this :)

Reply:
Agree that we'd probably want `cellToLocalIj`, but I think I skipped over it in the naming doc because of the experimental status. @isaacbrodsky, were we ever planning to graduate this out of "experimental"?

Reply:
Based on the last meeing, going to leave this one alone for this PR